README.md

```diff
@@ -41,6 +41,99 @@ root.mainloop()
 ```
 
 
+## Tips and tricks
+Sun Valley is meant to be a simple ttk theme wrapped in a PyPI package for convenience. I don't want to add other dependencies, nor features that would be trivial to implement if the user wants them, using 3rd party modules. But this theme can be easily integrated with them.
```

Comment:
```suggestion
Our intention is to keep the `sv-ttk` package as simple as possible, while making it easy to integrate with other libraries.
```

I don't like saying "I" or "the user" in places like this. Rather "we" and "you"